components/BarCard/index.js

```diff
@@ -3,25 +3,21 @@ import { Logo } from "../MatchCard";
 
 export default function BarCard({barWithMatch}){
     return(
-        <ListItem key={barWithMatch.id}>
+        <ListItem>
         <p>{barWithMatch.name}</p>
         <MatchPreviewSection>
-            {barWithMatch.matches.map((match) => (
-            <MatchPreview key={barWithMatch.id}>
-                <Logo key={match.homeTeam.name} style={{ backgroundColor: match.homeTeam.logoColor }} />
+            {barWithMatch.matches.map((match, index) => (
+            <MatchPreview key={`${match.id}-${index}`}>
+                <Logo style={{ backgroundColor: match.homeTeam.logoColor }} />
```

Comment:
We do prefer to not mix inline stylings and styled components. Please change this and create styled components for your inline stylings. 💪

Reply:
thanks, changed it! But I must say, that I agree in nearly all terms with using styled-components, but in this case I think the code is better readable when using inline-styling - that was the reason I used it :-)

Reply:
Ok, no all makes sense to me. Thanks for the explanation and the nice review :-)